index.html

```diff
@@ -361,6 +361,82 @@ <h3 class="skill-title" id="skill-title-3">
           </li>
         </ul>
       </section>
+      <section class="contact-section">
+        <p class="contact-paragraph">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          action="https://formspree.io/f/xrgwgyen"
+          method="post"
+          class="contact-form"
+        >
+          <input
+            type="text"
+            name="name"
+            class="form-input"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            class="form-input"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="comment"
+            id="text-comment"
+            cols="30"
+            rows="10"
+            placeholder="Write me something..."
+            required
+            maxlength="50"
+          ></textarea>
+          <input
+            type="submit"
+            name="submit-btn"
+            id="submit-btn"
+            value="Get in touch"
+            class="btn-get-in-touch"
+            required
+            maxlength="50"

```

Comment:
Nice job so far😉. It is a requirement that the message field accepts up to 500 characters and not 50😉. Kindly fix this😉.
<img width="298" alt="image" src="https://github.com/FilbertKimasa/Portfolio/assets/64643995/067017ed-b294-4cd3-8ed6-63cf944cb638">

```suggestion
            maxlength="500"

```